### PR TITLE
[ja] Translate `content/en/releases/notes.md` into Japanese

### LINKED ISSUE #44996: [ja] Translate `content/en/releases/notes.md` into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate `content/en/releases/notes.md` into Japanese

**Why is this needed**

There is no Japanese translation for this page

**Comments**

- File to translate: 
    - https://github.com/kubernetes/website/blob/main/content/en/releases/notes.md
- Original:
    - https://kubernetes.io/releases/notes/

ref. Japanese Localization document: https://kubernetes.io/ja/docs/contribute/localization/

/area localization
/language ja
/assign

### PR DESCRIPTION
**Pull request by Okabe-Junya**
close #44996 

## Change List

- Translate `content/en/releases/notes.md` into Japanese

### CODE REVIEW

**Comment by Okabe-Junya** [comment]
The target page on the deploy preview:

- https://deploy-preview-45001--kubernetes-io-main-staging.netlify.app/ja/releases/notes/

File: content/ja/releases/notes.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+description: >
+  Kubernetesのリリースノート
+sitemap:
+  priority: 0.5
+---
+
+リリースノートは、使用しているKubernetesのバージョンに合った[Changelog](https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG)を読むことで確認できます。
+{{< skew currentVersionAddMinor 0 >}}のchangelogを見るには[GitHub](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-{{< skew currentVersionAddMinor 0 >}}.md)を参照してください。
+
+またリリースノートは、[relnotes.k8s.io](https://relnotes.k8s.io)上で検索してフィルタリングすることもできます。
```
Comment (nasa9084) [inline content/ja/releases/notes.md:14]: great

**Review by nasa9084 — COMMENTED** [review]
/approve

**Pull request merged**